Source/Diagnostics/ReducedDiags/ChargeOnEB.cpp

```diff
@@ -120,9 +120,6 @@ void ChargeOnEB::ComputeDiags (const int step)
     amrex::Gpu::Buffer<amrex::Real> surface_integral({0.0_rt});
     amrex::Real* surface_integral_pointer = surface_integral.data();
 
-#ifdef AMREX_USE_OMP
-#pragma omp parallel if (amrex::Gpu::notInLaunchRegion())
-#endif
```

Comment:
LGTM, but can you add a comment here about a need to copy `surface_integral_pointer`  before adding parallelfor? this might reduce the risk to add it back in a cleanup.